db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -296,6 +297,9 @@ Status DBImpl::FlushMemTableToOutputFile(
                      flush_reason);
 
   bool switched_to_mempurge = false;
+  // If we exit a flush job since there is already a bg error,
+  // we should not set the bg error again.
+  bool skip_since_bg_error = false;
```

Comment:
Maybe call it ```skip_set_bg_error``` to make the purpose clearer.

Reply:
Yes, updated the name in FlushMemtableToOutputFile and FlushJob.